client/blocks/term-tree-selector/terms.jsx

```diff
@@ -41,7 +44,8 @@ const DEFAULT_TERMS_PER_PAGE = 100;
 const LOAD_OFFSET = 10;
 const ITEM_HEIGHT = 25;
 
-const TermTreeSelectorList = React.createClass( {
+const TermTreeSelectorList = createReactClass( {
```

Comment:
Hey @jsnmoon, would you have any idea why the codemod would substitute `createReactClass` for `React.createClass` instead of going ahead with the codemod? Just curious if I need to make another change first.

Reply:
Got it. Via your [react-create-class codemod PR](https://github.com/Automattic/wp-calypso/pull/13070):

>Ignore components that ... Explicitly call this.getInitialState()

https://github.com/reactjs/react-codemod/#explanation-of-the-new-es2015-class-transform-with-property-initializers